search-sentiments-map/public/scripts/modal-script.js

```diff
@@ -110,26 +110,29 @@ function displayTopResultsForCurrentTrend(countryCode) {
  * @param {string} countryCode Two letter country code for selected country.
  */
 function displaySentimentChartForCurrentTrend(countryCode) {
-  let countryData = getCurrentSearchData().dataByCountry.filter(data => 
-      data.country === countryCode);
+  let countryData = getCurrentSearchData().dataByCountry
+      .filter(data => data.country === countryCode);
   let date = new Date(getCurrentSearchData().timestamp);
   let chartElement = document.getElementById('sentiment-chart-tab');
   chartElement.innerHTML = '';
 
-  // Handle case where there are no results.
-  if (countryData.length === 0) {
+  // Handle case where there are no search results for the topic.
+  if (countryData.length === 0 ||
+      countryData[0].averageSentiment === NO_RESULTS_DEFAULT_SCORE) {
```

Comment:
Above(Around line 85) you have the comment in the if block, here it is before the if block. Make sure both are in the same place.

Reply:
I'd say put them in the if block bc it's for the first if clause

Reply:
I think you mean outside the block then?